src/ZfrCors/Factory/CorsServiceFactory.php

```diff
@@ -29,16 +28,17 @@
  * @license MIT
  * @author  Florent Blaison <florent.blaison@gmail.com>
  */
-class CorsServiceFactory implements FactoryInterface
+class CorsServiceFactory
 {
     /**
      * {@inheritDoc}
```

Comment:
@weierophinney I guess this is not good anymore, but what I should add here if typehint to `ContainerInterface` will be removed? `ServiceLocatorInterface|ContainerInterface`?